backend/plugins/frontline_api/src/modules/integrations/call/utils.ts

```
@@ -133,6 +122,14 @@ export const sendToGrandStream = async (models: IModels, args, user) => {
       return response;
     }
 
+    try {
+      const clonedRes = res.clone();
+      const maybeError = await clonedRes.json();
+      if (maybeError?.status === -6) {
+        return await retryWithFreshCookie();
+      }
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)